pgen2/pnode.py

```diff
@@ -27,3 +27,7 @@ def __repr__(self):
     ch_str = 'with %d children' % len(self.children) \
         if self.children is not None else ''
     return '(PNode %s %s %s)' % (self.typ, tok_str, ch_str)
+
+def NewPNode(typ, tok, children):
```

Comment:
Maybe write a comment about how this is rewritten in C++

I don't really understand that part

Reply:
Ah sure, I can add a note about this. This is here as an attempt to avoid GC pressure from all the PNodes. What confused you?

Reply:
Ah actually I think I see it. The list of child pointers is still managed by the GC, so the arena for the PNodes themselves is kinda useless...

Reply:
I'll push a commit to just translate PNode with mycpp. Then we can profile and optimize